common/src/Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet/tests/Default/SchemaValidationTests.cs

```diff
@@ -8,24 +8,64 @@ namespace Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet.Tests.Default {
     using System.IO;
     using System.Linq;
     using System.Text;
+    using System.Text.Json;
     using Xunit;
 
     /// <summary>
-    /// These tests are designed to exercise the 
-    /// generated schema for configuration file 
-    /// validation. These tests are additionally 
-    /// found in the schema generator tool located
+    /// These tests are designed to exercise the generated schema for configuration file 
+    /// validation. These tests are additionally found in the schema generator tool located
     /// at: https://github.com/WilliamBerryiii/opcpublisherschemavalidator
+    /// 
+    /// The referenced schema file is a linked asset in the project file set to 
+    /// copy to the output build directory so that it can be easily referenced here.
     /// </summary>
     public class SchemaValidationTests {
```

Comment:
Should we add one for Uri which was resulting in publisher hanging?

Reply:
absolutely. will resolve in next commit.

Reply:
Added.